packages/smooth_app/lib/pages/preferences/user_preferences_faq.dart

```diff
@@ -117,28 +117,26 @@ class UserPreferencesFaq extends AbstractUserPreferences {
             )
           ],
         ),
-        actions: <SmoothActionButton>[
-          SmoothActionButton(
-            onPressed: () async {
-              showLicensePage(
-                context: context,
-                applicationName: packageInfo.appName,
-                applicationVersion: packageInfo.version,
-                applicationIcon: Image.asset(
-                  'assets/app/smoothie-icon.1200x1200.png',
-                  height: MediaQuery.of(context).size.height * 0.1,
-                ),
-              );
-            },
-            text: appLocalizations.licenses,
-            minWidth: 100,
-          ),
-          SmoothActionButton(
-            onPressed: () => Navigator.of(context, rootNavigator: true).pop(),
-            text: appLocalizations.okay,
-            minWidth: 100,
-          ),
-        ],
+        positiveAction: SmoothActionButton(
+          onPressed: () async {
+            showLicensePage(
+              context: context,
+              applicationName: packageInfo.appName,
+              applicationVersion: packageInfo.version,
+              applicationIcon: Image.asset(
+                'assets/app/smoothie-icon.1200x1200.png',
+                height: MediaQuery.of(context).size.height * 0.1,
+              ),
+            );
+          },
+          text: appLocalizations.licenses,
+          minWidth: 100,
```

Comment:
I guess we could get rid of all those `minWidth: 100`, by handling it in `SmoothAlertDialog`.
It's not like "for this specific dialog, we have this specific need", it's just that in all cases we want a decent button size.